### ISSUE TITLE
2020

### ISSUE THREAD

**Issue by jw2249a** [Reed-EVIC/cpsvote#39]
Hi I didn't know if y'all started 2020 or not but I quickly updated the package for comparison if you did on my forked version. 

Also, I don't know why but the CPS started calling PES8 PRS8 in 2014, and I noticed that ur factor code relabels it as PES8 in 2018. It could honestly be a mistake by the Census/BLS ("E" is right next to "R" on the keyboard), but I relabeled them as PRS for years the Census published them as that. Other changes are just adding 2020 and the COVID questions.

**Comment by jw2249a** [comment]
Actually it appears the PES8/PRS8 issue is a mistake that has just been in census documentation since 2014 :/
![image](https://user-images.githubusercontent.com/22301712/127919506-7c92e971-09c2-44c1-8ed3-dacb8823f7b7.png)

**Comment by jw2249a** [comment]
Ahh PRS8 is apparently correct. 
![image](https://user-images.githubusercontent.com/22301712/128027753-3ab725db-4455-4331-b97b-b3f50d8fd964.png)

**Comment by jayleetx** [comment]
@jw2249a I've finished up the issues I was tackling, and am set to review/merge your additions now before version 0.2 goes off to CRAN. Could you put in a pull request from your fork to the [`add-2020` branch](https://github.com/Reed-EVIC/cpsvote/tree/add-2020)?

**Comment by kkprinceton** [comment]
Is there still a plan to add the 2020 version of this very helpful package to CRAN?

**Comment by paulgronke** [comment]
> Is there still a plan to add the 2020 version of this very helpful package to CRAN?

Yes, we have a new contributor added to the package and are going to clean up the file loading error we have been encountering in the 2020 branch and then integrate that branch. 

When we complete that process, we plan to move to add 2022 and 2024.

**Comment by paulgronke** [comment]
@jw2249a I think this issue has been resolved -- can you confirm? Is there anything else you'd like to consider adding to the "add-2020" branch before we merge and move on to completing work on 2022 and 2024?
